src/test/java/pages/LoginPage.java

```diff
@@ -3,36 +3,41 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 
 public class LoginPage extends BasePage{
 
     //locators
-    By submitButtonLocator = By.cssSelector("[type='submit']");
-    By emailField = By.cssSelector("[type='email']");
-    By passwordField = By.cssSelector("[type='password']");
+    @FindBy(css= "[type='submit']")
+            WebElement submitButtonLocator;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example
    
```
    @FindBy(css = "[type='submit']")
    private WebElement submitButtonLocator;
```